pkg/ebpf/c/bpf_helpers_custom.h

```diff
@@ -2,13 +2,23 @@
 #define __BPF_HELPERS_CUSTOM__
 
 #include "bpf_cross_compile.h"
+#include "compiler.h"
```

Comment:
I don't think this include is needed any longer, correct?

Reply:
Nope, let me remove it quickly